infra/serverless/lambda-web-app-test/index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WEATHER API HOMEPAGE</title>
+  </head>
+
+  <body>
+    <h3>Get Weather Forecast for Next 7 Days</h3>
+    <form>
+      <label for="location_input">Enter Location e.g., 'London':</label><br />
+      <input type="text" id="location_input" name="location" /><br />
+      <label for="unit_type_input">Enter Unit Type e.g., 'metric':</label><br />
+      <input type="text" id="unit_type_input" name="unitGroup" /><br />
+      <label for="include_hours_input">Include Hours?:</label><br />
+      <select id="include_hours_input" name="include">
+        <option value="yes">Yes</option>
+        <option value="no">No</option></select
+      ><br /><br />
+      <button id="submit">Get Weather Data</button>
+    </form>
+    <h3>Weather Forecast - Next 7 Days</h3>
+
+    <div id="response_api">
+      <table id="response_table"></table>
+    </div>
+    <script>
+      const submit_button = document.getElementById("submit");
+
+      var url =
+        " https://ivikocphja.execute-api.us-east-1.amazonaws.com/prod/forecast-data?";
```

Comment:
Can we make the value of the url dynamic and not hard coded. We can set the value at an environment level like `export API_GATEWAY_URL="https://ivikocphja.execute-api.us-east-1.amazonaws.com/prod/forecast-data?"` and have the local html retrieve this value when deployed locally OR even retrieve the value from AWS itself.

The api gateway url is static for now but it could change. Moving to a dynamic approach as above should minimise the amount of places we'll need to update hardcoded values